index.html

```diff
@@ -1,7 +1,8 @@
-<!doctype html>
+<!DOCTYPE html>
```

Comment:
Don't see you replacing any non-semantic HTML tags with a semantic one, as in Question #2.